src/content/glossary/index.md

```diff
@@ -813,6 +974,10 @@ Short for "test network," a network used to simulate the behavior of the main Et
   Testnets
 </DocLink>
 
+### token {#token}
+
+A fungible virtual good that can be traded.
+
```

Comment:
Token's can be both fungible and non-fungible. Not sure I have a great definition here though, but wouldn't declare fungibility here by default

Reply:
yea good point! 
I guess "virtual goods that can be traded on a blockchain" is better?
@minimalsm opinion?

Reply:
or "A tradable virtual good defined in smart contracts on the Ethereum blockchain"

Reply:
Personally, I prefer 'digital' over 'virtual' and 'asset' over 'good' for maximum comprehenability.

"A tradable digital asset defined in smart contracts on the Ethereum Blockchain." 

(although, if you want to argue ether is a token, this 👇 might be better). I think either is fine. And anyone who starts talking about soulbound non-fungible tokens can eff off 😆

"A tradable digital asset on the Ethereum Blockchain."